spring-petclinic-microservices/spring-petclinic-customers-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,15 @@
+spring:
+  cloud:
+    azure:
+      profile:
+        tenant-id: ${AZURE_KEYVAULT_TENANTID}
+      credential:
+        client-id: ${AZURE_KEYVAULT_CLIENTID}
+        client-secret: ${AZURE_KEYVAULT_CLIENTKEY}
+      keyvault:
+        secret:
+          property-source[0]:
+            refresh-interval: 1800000
```

Comment:
The refresh interval now is a Duration type, and 1800000ms equals to 30min, which is the default value.